src/ALZ/Assets/alz-bicep-config/v0.14.0.config.json

```
@@ -159,7 +159,7 @@
             ],
             "Value": "",
             "DefaultValue": "alz",
-            "Valid": "^[a-zA-Z]{2,5}$"
+            "Valid": "^[a-zA-Z]{2,5}(-[a-zA-Z]{2,5})?$"
```

```suggestion
            "Valid": "^[a-zA-Z0-9]{2,10}(-[a-zA-Z0-9]{2,10})?$"
```
update to match the conditions in the module https://github.com/Azure/ALZ-Bicep/blob/main/infra-as-code/bicep/modules/managementGroups/managementGroups.bicep#L6-L9